listenbrainz/webserver/static/js/src/player-pages/PlayerPage.tsx

```diff
@@ -115,6 +117,30 @@ export default class PlayerPage extends React.Component<
     );
   }
 
+  savePlaylist = async () => {
+    const { currentUser } = this.context;
+    if (!currentUser || !currentUser.auth_token) {
```

Comment:
Not particularly important, but this could be shortened to `!f (!currentUser?.auth_token)`

Reply:
Done.